cmd/upgrade.go

```diff
@@ -102,6 +105,9 @@ func (uc *upgradeCmd) validate(cmd *cobra.Command) error {
 		uc.timeout = &timeout
 	}
 
+	cordonDrainTimeout := time.Duration(uc.cordonDrainTimeoutInMinutes) * time.Minute
+	uc.cordonDrainTimeout = &cordonDrainTimeout
```

Comment:
If both vm-timeout and cordon-drain-timeout are provided, I think we should validate to make sure vm-timeout >= cordon-drain-timeout. Otherwise I think it doesn't make sense. Want to know you'll thoughts before I add that check?

Reply:
makes sense to me